k8s/controllers/metrics/attributes.go

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ * SPDX-License-Identifier: MIT
+ */
+
+package metrics
+
+// Deployment gets common logging attributes for a deployment.
+func Deployment(
+	reconciliationType ReconciliationType,
+	reconciliationResult ReconciliationResult,
+	resourceType ResourceType,
+	operationStatus OperationStatus,
+	chartVersion string,
```

Comment:
What's chartVersion?

Reply:
It will be retrieved by env variable - CHART_VERSION which records {{ .Chart.Version }}. We want to differentiate the metrics per helm chart versions. 
https://github.com/msftcoderdjw/eclipse-symphony/blob/9b0734cb8412aeabcb4e481145a4b5c3fb4d3939/k8s/controllers/metrics/metrics.go#L88-L89

The change to set this env var will be included in the 4th merge code PR that @iwangjintian will submit soon. 
https://github.com/msftcoderdjw/eclipse-symphony/blob/85f9823708584b01e2d573ca81c0dfc5ed298d22/packages/helm/symphony/templates/env-configmap.yaml#L8